web/js/compare/util.js

```diff
@@ -0,0 +1,15 @@
+import util from '../util/util';
+const DEFAULT_COMPARE_OBJ = {};
+export function getCompareObjects(models) {
+  if (!models.layers.activeB) return DEFAULT_COMPARE_OBJ;
```

Comment:
Even though it is const, the object DEFAULT_COMPARE_OBJ contents are mutable. Better to create a new empty object each time?